GitHub Actions CI 테스트 실패 - MinIO 연결 오류

## 문제

GitHub Actions에서 테스트 실행 시 MinIO 서버에 연결할 수 없어 57개 테스트가 실패합니다.

## 원인

- `application-test.yml`에 MinIO 비활성화 설정이 없음
- 기본값 `minio.enabled: true`로 MinIO 연결 시도
- CI 환경에는 MinIO 서버가 없어서 `java.net.ConnectException` 발생

## 에러 로그

```
Caused by: software.amazon.awssdk.core.exception.SdkClientException
Caused by: org.apache.http.conn.HttpHostConnectException
Caused by: java.net.ConnectException
```

## 해결 방법

`src/test/resources/application-test.yml`에 다음 설정 추가:

```yaml
minio:
  enabled: false
```

--- a/src/test/resources/application-test.yml
+++ b/src/test/resources/application-test.yml
@@ -32,6 +32,9 @@ jwt:
 firebase:
   enabled: false
 
+minio:
+  enabled: false
+
 # Snowflake ID 생성기 설정 (Redis 미사용)
 snowflake:
   use-redis-allocation: false

